Diagnostic/test.settings

```diff
@@ -0,0 +1 @@
, "unit": "Seconds", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk transfers", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "transferspersecond", "counterSpecifier": "/builtin/disk/transferspersecond", "type": "builtin", "unit": "CountPerSecond", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk write guest OS", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "writebytespersecond", "counterSpecifier": "/builtin/disk/writebytespersecond", "type": "builtin", "unit": "BytesPerSecond", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk read time", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "averagereadtime", "counterSpecifier": "/builtin/disk/averagereadtime", "type": "builtin", "unit": "Seconds", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk write time", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "averagewritetime", "counterSpecifier": "/builtin/disk/averagewritetime", "type": "builtin", "unit": "Seconds", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk total bytes", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "bytespersecond", "counterSpecifier": "/builtin/disk/bytespersecond", "type": "builtin", "unit": "BytesPerSecond", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk reads", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "readspersecond", "counterSpecifier": "/builtin/disk/readspersecond", "type": "builtin", "unit": "CountPerSecond", "sampleRate": "PT15S"}, {"annotation": [{"displayName": "Disk queue length", "locale": "en-us"}], "class": "disk", "condition": "IsAggregate=TRUE", "counter": "averagediskqueuelength", "counterSpecifier": "/builtin/disk/averagediskqueuelength", "type": "builtin", "unit": "Count", "sampleRate": "PT15S"}]}}, "sampleRateInSeconds": 15}}, "protectedSettings": "MIICmwYJKoZIhvcNAQcDoIICjDCCAogCAQAxggFpMIIBZQIBADBNMDkxNzA1BgoJkiaJk/IsZAEZFidXaW5kb3dzIEF6dXJlIENSUCBDZXJ0aWZpY2F0ZSBHZW5lcmF0b3ICEF2q8a+rPWOCTbrYYrJiNPAwDQYJKoZIhvcNAQEBBQAEggEAi7PjaBKLbeuqvBsKGB7FBYSw8Ikxo3567DtFuXy+LkKU/Ms4DQZnnij3ubo4/LYx3qNeo5quS/niY61Z3uS1s8J70EPY9KHfbJXnzdzD7dc2fkNzw1ZROplmMwepJSWQEc//aYe5g7ZsBK3aOYlO2Z30DbampoSVKlb99bsi1wQc4pKTtLCJig/Le0D997T5MWnDfKzQUUrWye0vYJe64k9siMNDrsbIkQOcjsXq9MBh4SmyznYJ3HQ6IPeahqoVe3XdogAK7stCnWMSLmtW4rQQXEjNiwYWShVxHv7COd315zbDIflMWBTKMNh55Tltk+5kkbZLtmWQsG63fWXNyDCCARQGCSqGSIb3DQEHATAUBggqhkiG9w0DBwQIBwy3u4+xh0qAgfAt+5PVrEZ8PqIbzZ5bCq9fJgpnccYDId86uACOlyXNRDAqREHL28Zyzw3pzQ4kVCieJew931Gc1D80Oo1EZspjzAScK+qIQn4SG6sSF51SaKMb+aABggrOficPVf1tVJ2vxHAXpXpW31+Dm9JGqxrbpAGQ9BN9yCYuwt6NI5mwrO52j5tiFx7sfliD6vgF0S1zy24CrcOmchwcqWnn4BicXYRMPMtmxCtDEdzXyz8E5tDoIRmPdxpkhaRfldkXdvlzC7oqMzGfvL+ivec6z1GZNG/eVsCwmWDGfN3e+A4PW04Lj/jBNm4daShHUCDPvrg=", "protectedSettingsCertThumbprint": "7125DD096E0770BC8B90A54445E55111ADD8C0EC"}}]}
```

Comment:
uhhHH might not wanna expose secrets here

Reply:
Ah good catch. removed it in the latest commit.